spec/event_spec.rb

```diff
@@ -0,0 +1,113 @@
+      @sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+      @knitting = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+      @event = Event.new("Carla's Craft Connection", [@sewing, @knitting], [@hector, @toni])
+      expect(@event.attendee_names).to eq(["Hector", "Toni"])
+    end
+  end
+
+  describe '#craft_with_most_supplies' do
+     it 'returns the craft with the most needed supplies' do
+       @sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+       @knitting = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+       @event = Event.new("Carla's Craft Connection", [@sewing, @knitting], [@hector, @toni])
+       expect(@event.craft_with_most_supplies).to eq("sewing")
+     end
+  end
+
+  describe '#supply_list' do
+    it 'returns the list of all supplies needed' do
+      @sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+      @knitting = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+      @event = Event.new("Carla's Craft Connection", [@sewing, @knitting], [@hector, @toni])
+      expected = ["fabric", "scissors", "thread", "sewing_needles", "yarn", "knitting_needles"]
+      expect(@event.supply_list).to eq(expected)
+    end
+  end
+
+  describe '#can_build?' do
+    it 'returns a boolean response if there are enough supplies to do the craft' do
+      @hector = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+      @sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+      @knitting = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+      @event = Event.new("Carla's Craft Connection", [@sewing, @knitting], [@hector, @toni])
+      expected = ["fabric", "scissors", "thread", "sewing_needles", "yarn", "knitting_needles"]
+      @hector.add_supply('fabric', 7)
+      @hector.add_supply('thread', 1)
+      @hector.add_supply('scissors', 1)
+      @hector.add_supply('sewing_needles', 1)
+      expect(@event.can_build?(@sewing)).to be true
+    end
+  end
+
+  xdescribe '#atendees_by_craft_interest' do
+    it 'returns a boolean response if there are enough supplies to do the craft' do
+      @hector = Person.new({name: 'Hector', interests: ['sewing', 'millinery', 'drawing']})
+      @sewing = Craft.new('sewing', {fabric: 5, scissors: 1, thread: 1, sewing_needles: 1})
+      @knitting = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+      @event = Event.new("Carla's Craft Connection", [@sewing, @knitting], [@hector, @toni])
+      expected = ["fabric", "scissors", "thread", "sewing_needles", "yarn", "knitting_needles"]
+      @hector.add_supply('fabric', 7)
+      @hector.add_supply('thread', 1)
+      @hector.add_supply('scissors', 1)
+      @hector.add_supply('sewing_needles', 1)
+      expect(@event.attendees_by_craft_interest).to eq()
+    end
+  end
```

Comment:
Would have like to see you finish the test before attempting to write the method to keep with TDD practices